js/command.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import * as msg from "gen/msg_generated";
+import * as flatbuffers from "./flatbuffers";
+import * as dispatch from "./dispatch";
+import { assert, unreachable } from "./util";
+
+export interface CommandOptions {
+  argv: string[];
+  dir?: string;
+  shell?: boolean;
+}
+
+interface ExitStatusCode {
+  success: boolean;
+  code: number;
+}
+
+interface ExitStatusSignal {
+  success: false;
+  signal: number /* TODO: Make this a string. */;
```

Comment:
I suspect prettier inserted the semicolon after the comment on formatting, but it looks odd here.